templates/mongoose_template.js.erb

```diff
@@ -1,12 +1,13 @@
 var Schema = mongoose.Schema;
 
-var <%= datatype %>Schema = new Schema({
-  <% for attribute in attributes %><% if attributes.last[:name] == attribute[:name] %>
-      <%= attribute[:name] %>: <%= MODELINFO_JS_LOOKUP[attribute[:type]] %>
-    <% else %>
-      <%= attribute[:name] %>: <%= MODELINFO_JS_LOOKUP[attribute[:type]] %>,
-    <% end %>
-  <% end %>
+var <%= datatype -%>Schema = new Schema({
+  <%- for attribute in attributes -%>
+  <%- if attributes.last[:name] == attribute[:name] -%>
+  <%= attribute[:name] %>: <%= MODELINFO_JS_LOOKUP[attribute[:type]] -%>
+  <%- else -%>
+  <%= attribute[:name] %>: <%= MODELINFO_JS_LOOKUP[attribute[:type]] -%>,
+  <%- end -%>
+  <%- end %>
```

Comment:
Any reason why all the indentation was removed in this block?

Reply:
Because the only lines that actually print anything are the `<%=` ones, and we only want a two space indentation, rather than a six.